cloud-service-provider/ibm/deployable-architectures/productivity-suite/terraform.tfvars

```diff
@@ -0,0 +1,80 @@
+# Example Terraform variables file
+# Copy this file to terraform.tfvars and populate with your actual values
+
+# IBM Cloud API key - Get from IBM Cloud console
+ibmcloud_api_key = "your-ibm-cloud-api-key-here"
+
+# Path to kubeconfig for IBM IKS cluster
+kubeconfig_file = "~/.kube/ibm-iks-kubeconfig"
+
+# API Tokens - Replace with your actual tokens
+hf_token       = "hf_your-huggingface-token-here"
+openai_api_key = "your-openai-api-key-or-jwt-token-here"
+
+# ChatQNA Configuration
+chatqna_model_dir = ""
+chatqna_helm_repo = "../../../../helm-charts/chatqna"
+
+chatqna_model_name           = "your-chatqna-model-name"
```

Comment:
Can we have a some value here, e.g. same as codegen and docsum are using since it is already deployed. Or smaller model like meta-llama/Llama-3.1-7B-Instruct